apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts

```diff
@@ -29,6 +30,7 @@ export class MembershipToggleComponent implements OnInit {
   @Input() status: NbComponentStatus = 'basic';
   @Input() size: NbComponentSize = 'small';
   @Input() appearance: NbButtonAppearance = 'filled';
+  joinCommunity = false;
```

Comment:
let's write data type!

Reply:
Type boolean trivially inferred from a boolean literal, remove type annotation.

It shows this error

Reply:
I have not deleted any previous function, just commented it out, once the testing is done, i will remove the function through which the component was working earlier